notebooks/retriever-expt.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<p>Add metrics to compare the results. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>